wgengine/netstack/netstack.go

```diff
@@ -190,8 +189,8 @@ func Create(logf logger.Logf, tundev *tstun.Wrapper, e wgengine.Engine, mc *magi
 	ipstack.SetPromiscuousMode(nicID, true)
 	// Add IPv4 and IPv6 default routes, so all incoming packets from the Tailscale side
 	// are handled by the one fake NIC we use.
-	ipv4Subnet, _ := tcpip.NewSubnet(tcpip.Address(strings.Repeat("\x00", 4)), tcpip.AddressMask(strings.Repeat("\x00", 4)))
-	ipv6Subnet, _ := tcpip.NewSubnet(tcpip.Address(strings.Repeat("\x00", 16)), tcpip.AddressMask(strings.Repeat("\x00", 16)))
+	ipv4Subnet, _ := tcpip.NewSubnet(tcpip.AddrFromSlice(bytes.Repeat([]byte{0x00}, 4)), tcpip.MaskFromBytes(bytes.Repeat([]byte{0x00}, 4)))
```

Comment:
might be easier to not use bytes.Repeat now and just write []byte{0,0,0,0} in both places

Reply:
I am not sure `[]byte{0,0,0,0}` would also work for `ipv6Subnet`? Somewhat `[]byte{0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0}` feels worse than bytes.Repeat.

Reply:
ping?